scripts/hf2bb.sl

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+# script version: slurm
+#
+# must be run with sbatch hf2bb.sl [realisationDirectory] [managementDBLocation]
+
+#SBATCH --cluster=Mahuika
+#SBATCH --job-name=hf2bb
+#SBATCH --account=nesi00213
+#SBATCH --partition=large
+#SBATCH --time=00:05:00
+#SBATCH --cpus-per-task=40
+
+REL_LOC=$1
+MGMT_DB_LOC=$2
+
+REL_NAME=`basename $REL`
+
+HF_LOC=$1/HF/Acc/HF.bin
+BB_LOC=$1/BB/Acc/BB.bin
+
+#updating the stats in managementDB
+if [[ ! -d $MGMT_DB_LOC/mgmt_db_queue ]]; then
+    #create the queue folder if not exist
+    mkdir $MGMT_DB_LOC/mgmt_db_queue
+fi
+timestamp=`date +%Y%m%d_%H%M%S`
+python $gmsim/workflow/scripts/cybershake/add_to_mgmt_queue.py $MGMT_DB_LOC/mgmt_db_queue $REL_NAME BB running
+
+runtime_fmt="%Y-%m-%d_%H:%M:%S"
+start_time=`date +$runtime_fmt`
+echo $start_time
+
+echo "python $gmsim/workflow/scripts/hf2bb.py $HF_LOC $BB_LOC"
+python $gmsim/workflow/scripts/hf2bb.py $HF_LOC $BB_LOC
+
+end_time=`date +$runtime_fmt`
+echo $end_time
+
+timestamp=`date +%Y%m%d_%H%M%S`
+#test before update
+res=`$gmsim/workflow/scripts/test_bb.sh $REL_LOC `
+if [[ $? == 0 ]]; then
+    #passed
+    python $gmsim/workflow/scripts/cybershake/add_to_mgmt_queue.py $MGMT_DB_LOC/mgmt_db_queue $REL_NAME BB completed
```

Comment:
I think there is a separate task for creating new tasks for these in the mgmt db along with alternative dependancies for BB.

Reply:
So between this task and that one, these scripts should have entries in the qcore process type enum, with associated MgmtDB entries and autosubmit logic?